stub/src/main/java/io/grpc/stub/ClientCalls.java

```diff
@@ -58,8 +58,8 @@
 
   @VisibleForTesting
   static boolean rejectRunnableOnExecutor =
-      !Strings.isNullOrEmpty(System.getenv("GRPC_CLIENT_CALL_REJECT_RUNNABLE"))
```

Comment:
Let's not enable this quite yet. I'd like us to take a look this first: https://github.com/grpc/grpc-java/blob/3fe2f8e4ff2a1b3d14bb208c706bd4a955681db8/stub/src/main/java/io/grpc/stub/ClientCalls.java#L165-L171

I think we'll likely want to avoid the shutdown() in those cases. But we'll see.